Modules/Remote/WebAssemblyInterface.remote.cmake

```diff
@@ -47,5 +47,5 @@ itk_fetch_module(
   "The itk-wasm WebAssemblyInterface module provides tools to a) build C/C++ code to WebAssembly-compatible processing pipelines, b) bridge local filesystems, JavaScript/Typescript data structures, and traditional file formats, c) transfer data efficiently in and out of the WebAssembly runtime."
   MODULE_COMPLIANCE_LEVEL 3
   GIT_REPOSITORY https://github.com/InsightSoftwareConsortium/itk-wasm.git
-  GIT_TAG 2f36f8df55f5b93b176d84670773cee10e3e1520
+  GIT_TAG 5babcf68be5fdfdd689dce97779d2085103c0bc1
```

Comment:
Don't know if it matters, but Matt just did another commit! https://github.com/InsightSoftwareConsortium/ITK-Wasm/commit/60223904ebb3f385c8e572defd5d8f3af97fa28a

Reply:
That can be updated in a follow-up PR. Also, it does not seem like an important change.